joomla/joomla-cms
# [4.0] proper html fallback everywhere

Pull Request for Issue #18764  .

### Summary of Changes
All index.php fallback to the same error page in case of unsupported PHP version. This should cover the cases @brianteeman mentioned in the issue:
- downgrade php (perhaps by moving host)
- use a hosting panel install script on an incompatible server (yes hosts are that stupid)


### Testing Instructions
You need to edit each of the index.php files (root, installation, administrator) and change the line
```php
if (version_compare(PHP_VERSION, JOOMLA_MINIMUM_PHP, '<'))
```
to 
```php
if (version_compare(PHP_VERSION, JOOMLA_MINIMUM_PHP, '>'))
```
and then navigate to 
`/`
`/installation/`
`/administrator/`

or any other URI pointing in your server. All cases should respond with the nice html page.

### Expected result



### Actual result



### Documentation Changes Required

Same as before, the js file for the translation is created through the gruntfile and the installation folder, so **maintainers** need to run the `grunt installation` to refresh that file if there are some changes in the languages files.
